### ISSUE TITLE
a problem of the code

### ISSUE THREAD

**Issue by wangyf8848** [PengtaoJiang/L2G#10]
Excuse me. What is the folder named SegmentationOnClassAug? And what is used to calculate the mIou with 'Attentions' in the test stage. I think 'Attentions' are evaluated by the number of 1464 segmentation labeled pictures in the voc12 at the beginning. BUT the code do not match my thought seamingly.

**Comment by SakaRenko** [comment]
Sorry for the late reply.
The folder named "SegmentationOnClassAug" contains the segmentation labels for the augmented version of Pascal VOC 2012, which contains 10582 pictures as the training set and 1449 pictures as the validation set. 
I think you may run the code on the another version of voc12 which only have a training set with 1464 pictures. 
To get the correct dataset, you can download from [here](https://drive.google.com/file/d/1uh5bWXvLOpE-WZUUtO77uwCB4Qnh6d7X/view)

**Comment by wangyf8848** [comment]
ok, thank you!
